### PR TITLE
new definitions for issue endpoint 

### PR DESCRIPTION
**Pull request by brandonbryant12**

### CODE REVIEW

File: src/handcash_minter.ts [hunk 1]
```diff
@@ -55,8 +53,12 @@ export default class HandCashMinter {
 	 * returns {Promise<CreateOrderItemResult[]}
 	 *
 	 * */
-	async createItems(params: CreateItemsParams): Promise<CreationOrderResult> {
-		return this.handCashConnectService.create(params);
+	async createItems(referencedCollection: string, items: CreateItemMetadata[]): Promise<CreateItemsOrder> {
```
Comment (rafa-js) [inline src/handcash_minter.ts:56]: Better as named parameters for autocompletion, etc...

**Review by rafa-js — COMMENTED** [review]
Makes more sense 👌 

Do we want to reconsider `user` in the item? `owner` or `userOwner`?

**Comment by brandonbryant12** [comment]
> Makes more sense 👌
> 
> Do we want to reconsider `user` in the item? `owner` or `userOwner`?

not sure it would be that different, `owner` is not used anywhere else so im hesitant to add a new name

**Review by rafa-js — APPROVED** [review]
LGTM

**Pull request merged**